components/OrchestratorList/index.tsx

```diff
@@ -69,6 +72,100 @@ const formatFactors = (factors: ROIFactors) =>
     ? `LPT Only`
     : `ETH Only`;
 
+const AvatarWrapper = ({
```

Comment:
@ECWireless, @Jipperism I didn't add a docstring since I expect we discuss and add this during milestone 4.